README.md

```diff
@@ -272,6 +272,12 @@ Restore state functionality is provided via a subset of entities:
 
 If you disable these entities, it is possible that various other entities may not be restored after restarting Home Assistant.
 
+##### Zigbee2MQTT
+
+While supported by the Blue switch firmware, Zigbee2MQTT does not yet provide events with an `action` for when a notification expires (it only, supports for instance [`config_double`](https://www.zigbee2mqtt.io/devices/VZM35-SN.html#action-enum)).
```

Comment:
What's the specific thing that needs to be populated? We can get that added to Z2M fairly easily.

Reply:
How is that made visible in ZHA/Home Assistant? Does it show up as an event or a new entity?

Reply:
It's just one of the many events you can consume of type `zha_event`.

Reply:
Alright, let me talk to Eric from Inovelli so I understand how these are intended to be used this evening and then see if I can get that implemented.

Reply:
Alright, I understand how this is supposed to be used, chatting with the Z2M devs since I can't find docs on unsolicited client to server reports in their code base. Should have something to test by Friday.

Reply:
This took a lot longer to get working. https://github.com/Koenkk/zigbee-herdsman-converters/pull/9735 adds support for it. It will be available via a new attribute called `notificationComplete `.

Reply:
Perfect. I'm out of town right now, but I'll try to incorporate it when I get back and have an update soon.